deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml

```diff
@@ -47,27 +47,27 @@ spec:
               - resourceKind
               type: object
             backingServiceSelector:
-              description: "BackingServiceSelector is used to identify the backing
-                service operator.  Refer: https://12factor.net/backing-services A
-                backing service is any service the app consumes over the network as
-                part of its normal operation. Examples include datastores (such as
-                MySQL or CouchDB), messaging/queueing systems (such as RabbitMQ or
-                Beanstalkd), SMTP services for outbound email (such as Postfix), and
-                caching systems (such as Memcached).  Example 1: \tbackingServiceSelector:
-                \t\tresourceType: database.example.org      resourceRef: mysql-database
-                Example 2: \tbackingServiceSelector: \t\tresourceType: database.example.org
-                \t\tresourceVersion: v1alpha1      resourceRef: mysql-database"
+              description: "BackingServiceSelector is used to identify the backing service
+                operator.  Refer: https://12factor.net/backing-services A backing
+                service is any service the app consumes over the network as part of
+                its normal operation. Examples include datastores (such as MySQL or
+                CouchDB), messaging/queueing systems (such as RabbitMQ or Beanstalkd),
+                SMTP services for outbound email (such as Postfix), and caching systems
+                (such as Memcached).  Example 1: \tbackingServiceSelector: \t\tresourceName:
+                database.example.org      objectName: mysql-database
+                Example 2: \tbackingServiceSelector: \t\tresourceName: database.example.org\t\t
+                resourceVersion: v1alpha1\t\tobjectName: mysql-database"
               properties:
-                resourceKind:
+                objectName:
                   type: string
-                resourceRef:
+                resourceName:
                   type: string
                 resourceVersion:
                   type: string
               required:
-              - resourceKind
+              - resourceName
               - resourceVersion
-              - resourceRef
+              - objectName
```

Comment:
This looks like an incorrect change.

Reply:
Indeed, that's misplaced, reverting this change.